launch-game.sh

```diff
@@ -25,7 +25,16 @@ mono OpenRA.Game.exe Engine.LaunchPath="$MODLAUNCHER" $MODARG "$@"
 
 # Show a crash dialog if something went wrong
 if [ $? != 0 ] && [ $? != 1 ]; then
-	ERROR_MESSAGE="OpenRA has encountered a fatal error.\nPlease refer to the crash logs and FAQ for more information.\n\nLog files are located in ~/.openra/Logs\nThe FAQ is available at http://wiki.openra.net/FAQ"
+	if [ "$OSTYPE" == "darwin"* ]; then
```

Comment:
Bleh, shellcheck doesn't like a lot of this :(
```
Line 28:
        if [ "$OSTYPE" == "darwin"* ]; then
              ^-- SC2039: In POSIX sh, OSTYPE is undefined.
                       ^-- SC2039: In POSIX sh, == in place of = is undefined.
                          ^-- SC2081: [ .. ] can't match globs. Use a case statement.
 
Line 29:
                LOGS="~/Library/Application Support/OpenRA/Logs/"
                      ^-- SC2088: Tilde does not expand in quotes. Use $HOME.
 
Line 32:
                if [ ! -d "${LOGS}" && -d "${HOME}/.openra/Logs" ]; then
                                    ^-- SC2107: Instead of [ a && b ], use [ a ] && [ b ].
 ```